docs/source/en/quantization.md

```diff
@@ -518,6 +518,19 @@ double_quant_config = BitsAndBytesConfig(
 model_double_quant = AutoModelForCausalLM.from_pretrained("meta-llama/Llama-2-13b", quantization_config=double_quant_config)
 ```
 
+## AQLM
```

Comment:
Would it be possible to move that on the top of the doc page before AWQ? we sort them in the integration order

Reply:
Done. I also added a short description of available inference setups.

Reply:
Awesome thanks !